### ISSUE TITLE
No "stats" folder in "fsl_gamma_basis"

### ISSUE THREAD

**Issue by cmaumet** [incf-nidash/nidmresults-examples#7]
Hi @AlexBowring,

It looks like the `stats` folder is missing in `fsl_gamma_basis`. Could you have a look and upload if you have it locally? Thank you!

**Comment by AlexBowring** [comment]
Ah @cmaumet, i didn't spot it when I first did it but looking at the report now there was actually an error in the analysis, which explains why it didn't write the file. I tried running the analysis again but am getting errors (although different ones this time) - can't see anything obvious in the .fsf file, we may have to wait until you return on this one and look then.

**Comment by nicholst** [comment]
Send me the errors and even better zip up the feat folder and I'll see if I can help. 

-Tom

**Comment by nicholst** [comment]
Hi @AlexBowring, I've found the problem... 

```
/usr/local/packages/fsl-5.0.8/bin/epi_reg --epi=example_func --t1=highres_head --t1brain=highres --out=example_func2highres
Image Exception : #99 :: Out of memory
```

What machine were you running on?  If it is your own Mac, some things to try is to make sure every other program is quite (especially web browsers or any Microsoft programs).  Also,  you can save even more memory by killing the Finder... in a Terminal window do:

```
defaults write com.apple.finder QuitMenuItem -bool YES
```

which will give you a new menu item "Quit" under the "Finder" menu; but one-time only, you'll need to enter the `killall Finder` command in the Terminal.  When you need the Finder after quitting it, you just need to click on the Finder icon in the Dock.

**Comment by nicholst** [comment]
Oh, and @AlexBowring if you're working on Buster... you may have just had bad luck and your host ran out of memory due to too many people running big mem jobs.  Delete and try running again and see if you still get the error.

**Comment by AlexBowring** [comment]
@nicholst I am working on Buster, given this a few goes now but getting the same Out of Memory error every time!

**Comment by cmaumet** [comment]
This has been corrected in #14.
